README.md

```diff
@@ -1,13 +1,13 @@
 # Towa GDPR Plugin
 
 This Plugin is a cookie notice plugin conforming with the GDPR regulation, implemented by the EU. 
```

Comment:
@Martin-Welte Could you link "GDPR"?
https://eur-lex.europa.eu/eli/reg/2016/679/oj (down at the moment)